src/object/obj_verify.c

```diff
@@ -807,12 +807,14 @@ dc_obj_verify_ec_rdg(struct dc_object *obj, struct dc_obj_verify_args *dova,
 				D_GOTO(out, rc);
 			}
 
-			rc = dc_obj_enum_unpack(oid, dova->kds, dova->num, &dova->list_sgl,
-						NULL, dc_obj_verify_ec_cb, dova);
-			if (rc) {
-				D_ERROR(DF_UOID" failed to verify ec object: "DF_RC"\n",
-					DP_UOID(oid), DP_RC(rc));
-				D_GOTO(out, rc);
+			if (dova->non_exist && dova->num > 0) {
```

Comment:
Should be "if (!dova->non_exist && dova->num > 0)"?

Reply:
ah, right. thanks!